helm/minio/values.yaml

```diff
@@ -422,6 +422,42 @@ buckets: []
   #   # set objectlocking for
   #   # bucket [true|false] NOTE: versioning is enabled by default if you use locking
   #   objectlocking: false
+  # - name: bucket3
+  #   # A policy of 'custom' allows you to set a custom JSON formatted policy in policyJson
+  #   # that is applied to the bucket via 'mc anonymous set-json'
+  #   policy: custom
+  #   policyJson: |-
+  #     {
+  #         "Version": "2012-10-17",
+  #         "Statement": [
+  #             {
+  #                 "Effect": "Allow",
+  #                 "Principal": {
+  #                     "AWS": [
+  #                         "*"
+  #                     ]
+  #                 },
+  #                 "Action": [
+  #                     "s3:GetObject",
+  #                     "s3:ListBucket"
+  #                 ],
+  #                 "Resource": [
+  #                     "arn:aws:s3:::bucket3",
+  #                     "arn:aws:s3:::bucket3/*"
+  #                 ],
+  #                 "Condition": {
+  #                     "IpAddress": {
+  #                         "aws:SourceIp": [
+  #                             "10.42.0.0/16",
+  #                         ]
+  #                     }
+  #                 }
+  #             }
+  #         ]
+  #     }
+  #   purge: false
+  #   versioning: true
+  #   objectlocking: false
```

Comment:
this is already referenced at line #424 - we do not need double references.

Reply:
Indeed, removed the extra attributes on bucket3 to only demonstrate the custom policy attributes.